### PR TITLE
Thu/dataset overview prod 12475

### PR DESCRIPTION
**Pull request by csm-thu**
PR ready for review

### CODE REVIEW

File: src/state/sagas/datasets/QueryDatasetTwingraph/QueryDatasetTwingraph.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { all, call, delay, put, select, spawn, takeEvery } from 'redux-saga/effects';
+import { Api } from '../../../../services/config/Api';
+import {
+  TWINGRAPH_QUERIES_DELAY,
+  TWINGRAPH_QUERY_MAX_RETRIES,
+  TWINGRAPH_QUERY_RETRY_DELAY,
+} from '../../../../services/config/FunctionalConstants';
+import { DATASET_TWINGRAPH_QUERIES_RESULTS_ACTIONS } from '../../../commons/DatasetConstants';
+
+const getOrganizationId = (state) => state.organization.current?.data?.id;
```
Comment (esasova) [inline src/state/sagas/datasets/QueryDatasetTwingraph/QueryDatasetTwingraph.js:13]: just out of curiosity: in other sagas, we passe `organizationId` by the hook, is there a special need to pass by` yield select `in this one or it's just a syntax choice? :slightly_smiling_face:

Comment (esasova) [inline src/state/sagas/datasets/QueryDatasetTwingraph/QueryDatasetTwingraph.js:13]: just out of curiosity: in other sagas, we passe `organizationId` by the hook, is there a special need to pass by` yield select `in this one or it's just a syntax choice? :slightly_smiling_face:

File: src/state/reducers/datasetTwingraphQueriesResults/DatasetTwingraphQueriesResultsReducer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        const expectedKpis = [...queriesMapping[queryId]];
+        result.data.forEach((result) => {
+          for (const [kpiId, kpiValue] of Object.entries(result)) {
+            state[datasetId][kpiId].value = kpiValue;
+            state[datasetId][kpiId].state = KPI_STATE.READY;
+            const kpiIndex = expectedKpis.indexOf(kpiId);
+            if (kpiIndex !== -1) expectedKpis.splice(kpiIndex, 1);
+          }
+        });
+        expectedKpis.forEach((missingKkpi) => (state[datasetId][missingKkpi].state = KPI_STATE.UNKNOWN));
```
Comment (esasova) [inline src/state/reducers/datasetTwingraphQueriesResults/DatasetTwingraphQueriesResultsReducer.js:42]: typo: _missingKpi_ (one K) :slightly_smiling_face:

Comment (csm-thu) [reply src/state/reducers/datasetTwingraphQueriesResults/DatasetTwingraphQueriesResultsReducer.js]: Fixed, thank you :slightly_smiling_face:

File: src/config/overrides/Workspaces.js [hunk 3]
```diff
@@ -123,88 +123,82 @@ export const WORKSPACES = [
...
-              type: 'relationship',
+              id: 'bars',
+              name: { en: 'Bars', fr: 'Bars' },
+              type: 'entity',
               description: {
                 en:
-                  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor  incididunt ut ' +
-                  'labore et dolore magna aliqua.',
+                  'Bars are compound entities in the Brewery model. They are responsible of the stock management and ' +
+                  'the number of waiters.\n An entity of type Bar is the parent enity of Customers inside this bar.',
```
Comment (esasova) [inline src/config/overrides/Workspaces.js:147]: typo: en**T**ity :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
Just one typo, LGTM :+1:

File: src/config/overrides/Workspaces.js (hunk 3 above)
Comment (csm-thu) [reply src/config/overrides/Workspaces.js]: Fixed, thank you ! :)

**Review by nborde-CSM — APPROVED** [review]
LGTM

**Pull request merged**